proto/src/main/java/org/dozer/util/ProtoUtils.java

```diff
@@ -66,18 +69,28 @@ private ProtoUtils() {
 
         Descriptors.FieldDescriptor foundDescriptor = null;
         for (Descriptors.FieldDescriptor descriptor : protoFieldDescriptors) {
-            if (fieldName.equals(descriptor.getName())) {
+            if (sameField(fieldName, descriptor.getName())) {
+                // XXX comment why return not done immediately at first match
```

Comment:
I put this comment because I was wondering why it wasn't done in the first place.  Is there a good reason?  Seems awkward to me.  I can add a `break` if you think likewise.